.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -83,7 +74,7 @@ body:
     label: ❌ Actual Behavior
     placeholder: What happened instead?
   validations:
-    required: false
+    required: true
```

Comment:
I think this can become duplicative which is why I had it false.

Reply:
Yeah, I've also seen people copy-pasting the exact same text three times. Thought this would bring som attention to the difference between "how to get there" versus "the actual result"...

Reply:
Revert please